common/util/rviz_plugins/autoware_perception_rviz_plugin/README.md

```diff
@@ -0,0 +1,43 @@
+# autoware_perception_rviz_plugin
+
+## Purpose
+This plugin is used to generate dummy pedestrians, cars, and obstacles in planning simulator.  
+
+## Overview
+The CarInitialPoseTool simulates a dummy car.  
```

Comment:
simulates a dummy car
->
sends a topic for generating dummy car ...??

Reply:
That one is more accurate :smile: 
fix https://github.com/tier4/AutowareArchitectureProposal.iv/pull/591/commits/99f41135cb7923ed2f93866145c42a38ee719065